services/src/datasets/external/aruna/mod.rs

```diff
@@ -178,8 +180,11 @@ impl ArunaDataProvider {
     /// Creates a new provider from the given definition.
     async fn new(def: Box<ArunaDataProviderDefinition>) -> Result<ArunaDataProvider> {
         let url = def.api_url;
+        let timeout = 10;
```

Comment:
Ideally, this would be a configurable field in the Provider Definition. I assume this would require a migration for the database. There were already some new feature requests the past couple of days (i.e., not having a dedicated geo engine project, ...), so I think it would be best to bundle that into a feature that changes the the Provider Definition anyway.